### ISSUE TITLE
Is there a way to get a smaller version?

### ISSUE THREAD

**Issue by tommytcchan** [albeebe/phoneformat.js#52]
I'm using webpack to build my project and I included this as my library. However this library add over 500+ KB to my project and that's a considerable size:

  [526] ./~/phoneformat.js/dist/phone-format-exports.js 507 kB {0} [built]

Is there a way to shrink the size of this library?

**Comment by sturdynut** [comment]
Not really, the bloat is from the google libraries, which are 478kb, even compressed they are still about 414kb.
